src/deploy/NVA_build/Base.Dockerfile

```diff
@@ -32,6 +32,7 @@ RUN ./src/deploy/NVA_build/clone_s3select_submodules.sh
 RUN ln -s /lib64/libboost_thread.so.1.66.0 /lib64/libboost_thread.so.1.75.0 || true
 #Pass BUILD_S3SELECT down to GYP native build.
 #S3Select will be built only if this parameter is equal to "1".
+#Next step would fail for RHEL8 and derivates
```

Comment:
Is this comment correct?
I thought the point is to fix RHEL8?

Reply:
Actually this dockerfile is not used for RPM generation so doesn't effect this PR. The reason I still left this comment here so that when someday someone tries to build a NooBaa image with Centos8 as base image and when it fails, it won't be a total surprise.

I am not sure if we want to fix it. This dockerfile is used for generating noobaa-core image and test images but none of them _need_ centos8/rhel8 as its base (while the same isn't true for RPMs, both scale and DBS3 need RPMs specifically generated for RHEL8).

The gist is, we should probably fix it iff we plan to keep on supporting Centos8 based docker/OCI images for NooBaa core, which I am not sure if we want to.

Reply:
OK, makes sense.
Can you just fix typo derivates -> derivatives